index.bs

```diff
@@ -7,10 +7,11 @@ Group: webtransport
 ED: https://w3c.github.io/webtransport/
 TR: https://www.w3.org/TR/webtransport/
 Editor: Bernard Aboba, w3cid 65611, Microsoft Corporation
-Editor: Victor Vasiliev, w3cid 113328,Google
-Editor: Yutaka Hirano, w3cid 100504,Google
+Editor: Nidhi Jaju, w3cid 136840, Google
+Editor: Victor Vasiliev, w3cid 113328, Google
```

Comment:
Editors don't appear to be sorted alphabetically. Should we preserve historical order? Or did @vasilvv ask to be moved?

Reply:
Alphabetical order probably makes more sense.

Reply:
Apologies! It seemed that the names were in alphabetical order by first name previously, but maybe that was just a coincidence. I'll leave it as is for now based on Victor's comment.